src/com_tjvendors/tjvendors.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <extension type="component" version="3.0" method="upgrade">
 	<name>COM_TJVENDORS</name>
-	<creationDate>26th Jul 2021</creationDate>
-	<copyright>Copyright (C) 2016 - 2021 Techjoomla. All rights reserved.</copyright>
+	<creationDate>25th Jan 2022</creationDate>
+	<copyright>Copyright (C) 2016 - 2022 Techjoomla. All rights reserved.</copyright>
 	<license>http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL</license>
 	<author>Techjoomla</author>
 	<authorEmail>extensions@techjoomla.com</authorEmail>
 	<authorUrl>https://techjoomla.com</authorUrl>
-	<version>1.4.3</version>
+	<version>2.0.0</version>
```

Comment:
@ankush-maherwal is sql file name changed to match version?

Reply:
yes, its already updated